Dockerfile.alpine.armv7hf

```diff
@@ -1,11 +1,22 @@
 FROM resin/armhf-alpine-node:0.10.44-slim
 
+ENV DOCKER_COMPOSE_VERSION 1.7.1
+ENV DOCKER_COMPOSE_SHA256 3f0b8c69c66a2daa5fbb0c127cb76ca95d7125827a9c43dd3c36f9bc2ed6e0e5
+
 # Supervisor apt dependencies
 RUN apk add --update \
 		btrfs-progs \
 		ca-certificates \
+		curl \
 		rsync \
 		supervisor \
+	&& curl -sLO http://resin-packages.s3.amazonaws.com/docker-compose/${DOCKER_COMPOSE_VERSION}/docker-compose-linux-armhf-${DOCKER_COMPOSE_VERSION}.tar.gz \
```

Comment:
@pcarranzav this docker-compose binary will not work on Alpine since Alpine use musl-libc instead of glibc on Debian. I will create Alpine version asap